superset/tasks/thumbnails.py

```diff
@@ -49,6 +49,7 @@ def cache_chart_thumbnail(
             user = security_manager.get_user_by_username(
                 current_app.config["THUMBNAIL_SELENIUM_USER"], session=session
             )
+        user = session.merge(user)
```

Comment:
possible that the user is not bound to a session since it's outside of the `nullpool` session block and at the end of the block the session is closed, can you revert and test placing `screenshot.compute_and_cache` under the `session_scope`?

Reply:
Yeah it worked as well. I changed code accordingly.

Reply:
Awesome!! thank you for the fix